### PR TITLE
Configurable maximum cache size

### PR DESCRIPTION
**Pull request by beaverking1212**
Configurable maximum cache size with 8 GiB default.

### CODE REVIEW

**Review by kingosticks — CHANGES_REQUESTED** [review]

File: README.rst [hunk 1]
```diff
@@ -169,6 +169,8 @@ The following configuration values are available:
 - ``spotify/allow_cache``: Whether to allow caching. The cache is stored in a
   "spotify" directory within Mopidy's ``core/cache_dir``. Defaults to ``true``.
 
+- ``spotify/cache_size``: maximum cache size in MiB. Defaults to ``8192``.
```
[the remark this answers is not in the document]
Comment (beaverking1212) [reply README.rst:172]: I think the default should be a constant. Here is a table with approximate cache size in songs and hours. I think the everyday music library should fit in the cache for the cache to be efficient.
| Cache size | Approx. hours 320 kbps | Approx. songs 320 kbps | Approx. hours 160 kbps | Approx. songs 160 kbps | 
|-----|-----|-----|-----|-----|
| 8192 MB |59 h|889 songs|118 h|1778 songs|
| 4096 MB |30 h|444 songs|59 h|889 songs|
| 2048 MB |15 h|222 songs|30 h|444 songs|

Should the default be 4096, 2048 or some other size?

**Comment by beaverking1212** [comment]
Tests fixed and readme changed as requested.

**Review by kingosticks — APPROVED** [review]

**Comment by kingosticks** [comment]
Let's go with 8GB for now, there's no one-size fits all answer and it's clearly documented for adjusting. thanks!

**Comment by beaverking1212** [comment]
Thank you!

**Pull request merged**